linux_os/guide/system/network/network-nftables/nftables_rules_permanent/rule.yml

```diff
@@ -17,7 +17,7 @@ rationale: |-
 
 severity: medium
 
-platform: package[nftables]
+platform: package[nftables] and not package[firewalld]
```

Comment:
Same case of `nftables_ensure_default_deny_policy` rule.

Reply:
Same motivation as for default_deny_policy rule